cmd/servicelog/post.go

```diff
@@ -72,6 +72,9 @@ func newPostCmd() *cobra.Command {
 }
 
 func (o *PostCmdOptions) Init() error {
+	userParameterNames = []string{}
+	userParameterValues = []string{}
+	filterParams = []string{}
```

Comment:
@AlexVulaj this is what I mentioned to you earlier. We had to do some shenanigans to send multiple service logs in the same execution.

Reply:
All the more reason I want us to start getting rid of global/package scoped vars as we come across them 😞